collection_modules/btle_beacon/moduleConfigLoader.py

```diff
@@ -40,6 +40,38 @@ def loadModule(thisConfig, logger, configParser):
 
     exit
 
+    """btel_test_mode"""
```

Comment:
again all this existed in the last PR that i had that took a couple days to get done.